### PR TITLE
Add galoy label in charts PR

### PR DESCRIPTION
**Pull request by sandipndev**

### CODE REVIEW

**Comment by bodymindarts** [comment]
Before this can be merged the label has to be added to the charts repo manually or the job will fail. Has this been done @sandipndev ?

**Comment by sandipndev** [comment]
Yes, the "galoy" label has been added manually.

**Review by bodymindarts — APPROVED** [review]

**Review by krtk6160 — APPROVED** [review]

File: ci/tasks/open-charts-pr.sh [hunk 1]
```diff
@@ -30,3 +30,4 @@ gh pr create \
   --base ${BRANCH} \
   --head ${BOT_BRANCH} \
   --label galoybot
+  --label galoy
```
Comment (bodymindarts) [inline ci/tasks/open-charts-pr.sh:33]: You are missing a `\` at the end of the line above. In bash `\` means that the command continues on the next line.

**Review by bodymindarts — CHANGES_REQUESTED** [review]

File: ci/tasks/open-charts-pr.sh (hunk 1 above)
Comment (sandipndev) [reply ci/tasks/open-charts-pr.sh:33]: OMG yes, thanks for noticing this!

**Review by bodymindarts — APPROVED** [review]

**Review by krtk6160 — APPROVED** [review]

**Pull request merged**